apps/api/src/queries/term/term-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -131,9 +133,12 @@ const assertResourceHasContributionFor = (
  * TODO We need to find a more maintainable way of
  * seeding the required initial state.
  */
-const buildEventHandlers = (termQueryRepository: ITermQueryRepository) => {
+const buildEventHandlers = (
+    termQueryRepository: ITermQueryRepository,
+    commandInfoService: CommandInfoService
+) => {
     return [
-        new TermCreatedEventHandler(termQueryRepository),
+        new TermCreatedEventHandler(termQueryRepository, commandInfoService),
```

Comment:
Is the second dependency still needed? Isnt' it now a dependency of the query service given that the events only update the actions by name (not form \ schema)?

Reply:
Removed.